src/Container/Definition/ClassDefinition.php

```diff
@@ -271,14 +272,17 @@ public function compile(
             if (array_key_exists("class", $constructorArgument)) {
                 $definition = $compilation->getDefinition($constructorArgument["class"]);
 
-                $code .= "\n{$constructorIndent}{$tab}" . $this->compileEntryReference(
+                $code .= $this->indentLines("{$constructorIndent}{$tab}", $this->compileEntryReference(
```

Comment:
if you generate the example container with the `make build` command, you'll see that this makes the indentation of the dependencies of `PlantService` wrong. The rest of the changes look good though.

Reply:
Fixed 🍏 

Is is all right now?

Reply:
I don't understand why this change is needed at all when the original indentation looked good for me?

Reply:
`var_export` has a fixed indentation of 2 spaces.
https://github.com/woohoolabs/zen/pull/20/commits/a1be5e79dbbb5228f180d3856f32e568d56af573#diff-735c9178f83fdb6831e51d7073bcee55372c3a5c83b4bed1edf08deffb7fe774
To correct that I've added this method.